index.html

```diff
@@ -0,0 +1,382 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+
+  <!-- fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <!-- custom css -->
+  <link rel="stylesheet" href="style.css" />
+
+  <!-- google font -->
+  <style>
+    @import url("https://fonts.googleapis.com/css2?family=Oswald:wght@400;500;600&family=Roboto:wght@300;400;500;700;900&display=swap");
+  </style>
+</head>
+
+<body>
+  <div id="header">
+    <i class="fa-solid fa-bars"></i>
+    <a href="index.html"><i class="fa-brands fa-youtube"></i> <span>YouTube</span></a>
+    <input type="text" placeholder="Search" />
+    <button><i class="fa-solid fa-magnifying-glass"></i></button>
+
+    <i class="fa-solid fa-microphone"></i>
+
+    <i class="fa-regular fa-moon"></i>
+
+    <i class="fa-regular fa-bell"></i>
+
+    <img src="photo/profile.jpg" alt="picture" class="profile" />
+  </div>
+
+  <div id="container">
```

Comment:
Apart from the header section, I think it would be better to use the <section> tag to clear your code. 
What do you think about that? 🤔

Reply:
- [ ] Awesome inline comment! But I did not understand this ` I think it would be better to use the tag to clear your code.`, perhaps something is missing. Kindly clarify it further. 👍

Reply:
- [ ] Awesome inline comment! But I did not understand this ` I think it would be better to use the tag to clear your code.`, perhaps something is missing. Kindly clarify it further. 👍